docs/cudf/source/user_guide/performance_comparisons.ipynb

```diff
@@ -0,0 +1,1517 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        gd_time = timeit.timeit(lambda : func(gd_obj), **kwargs)
</pre><p>Can we call this <code>cudf_time</code>? And maybe <code>pd_time</code> should be <code>pandas_time</code> to align with the naming conventions below.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/12595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Done